### DISCUSSION TITLE
I am still cant verify my school id

### DISCUSSION THREAD

**Discussion by yagizballi** [community/community#54895] [GitHub Education]
### Hi everyone,

I have been trying to verify my student ID for weeks, but I still can't get it verified. I translated my ID from Turkish to English, but it says, "Your uploaded proof has already been used previously."

**Comment by iamreiyn** [comment]
Hi @yagizballi 

Ensure that your evidence matches the information on your GitHub profile. The verification process will check your GitHub username, email, name, and other relevant details to confirm the evidence submitted. To avoid differences, use the same name or nickname as written on your student ID. If you update your profile information, try logging out and logging back in before re-applying.

Let us know how your application goes!

**Comment by yagizballi** [comment]
i still get "Your uploaded proof has already been used previously." error. I tried logging out. I think my id is correct. I can share my id in here if u need.

**Reply by iamreiyn** [reply]
Hi, please do not share any of your personal info here, as this is the GitHub community and not managed by support (private), so anyone can see the information posted here. I would recommend trying different means to prove your academic status.

> When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
> 
> If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

[Source](https://github.com/orgs/community/discussions/54169)

**Reply by iamreiyn** [reply]
@yagizballi Also, try taking a picture of your translated ID alongside the original ID (or uploading a picture that has your translated and original ID combined)
